Libraries/src/Amazon.Lambda.DynamoDBEvents/DynamoDBTimeWindowResponse.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace Amazon.Lambda.DynamoDBEvents
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+
+#if !NETSTANDARD2_0
+    using Amazon.Lambda.DynamoDBEvents.Converters;
+    using System.Text.Json.Serialization;
+#endif
+
+    /// <summary>
+    /// Response type to return a new state for the time window and to report batch item failures.
+    /// </summary>
+    [DataContract]
+    public class DynamoDBTimeWindowResponse
+    {
+        /// <summary>
+        /// New state after processing a batch of records.
+        /// </summary>
+        [DataMember(Name = "state")]
+#if NETCOREAPP_3_1
+        [System.Text.Json.Serialization.JsonPropertyName("state")]
+#endif
+#if !NETSTANDARD2_0
+        [JsonConverter(typeof(DictionaryLongToStringJsonConverter))]
```

Comment:
This can be combined with the previous `NETCOREAPP_3_1` condition block.

Reply:
I have changed it to `NETCOREAPP3_1_OR_GREATER` (else it was failing test case) since test case was having this conditional compilation check. Tested locally.